lib/internal/Magento/Framework/App/Response/Http/FileFactory.php

```diff
@@ -48,13 +48,44 @@ public function __construct(
      * @SuppressWarnings(PHPMD.CyclomaticComplexity)
      * @SuppressWarnings(PHPMD.NPathComplexity)
      * @SuppressWarnings(PHPMD.ExitExpression)
+     *
+     * @deprecated
```

Comment:
Please mention the reason of deprecation according to the guide https://devdocs.magento.com/guides/v2.2/contributor-guide/backward-compatible-development/#php-js-and-xml